src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_help.py

```diff
@@ -0,0 +1,11 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+
+helps['backup'] = """
+            type: group
+            short-summary: Backup cmdlets.
```

Comment:
Using the word 'cmdlets' is inconsistent with the rest of the other commands.
Maybe 'Commands to manage Azure Backups' or something.

Reply:
Agreed. "cmdlets" is a Powershell-specific term not used anywhere in the CLI.